etc/vscode/README.md

```diff
@@ -0,0 +1,65 @@
+# pris README
```

Comment:
Maybe fill this file with a sentence or two about what the vscode plugin does, and how to get it working?